bom_csv_jlcpcb.py

```diff
@@ -26,7 +26,7 @@
 
 net = kicad_netlist_reader.netlist(sys.argv[1])
 
-with open(sys.argv[2], 'wb') as f:
+with open(sys.argv[2], 'w') as f:
```

Comment:
thanks, but removing the `b` breaks it on Windows and Python 2. Perhaps we can use `wb` for python 2 and `b` for python 3?

Reply:
Python 2 is end-of-life since this year, so personally I wouldn't worry about that :D
The combination Python3+Windows is also not happy with 'w' only?

Reply:
Seems like for python 3 it requires the newline parameter:

Reply:
https://stackoverflow.com/a/3348664/830623

Reply:
`newline=''` doesn't break it on my linux box at least. Can't test on Windows unfortunately.

Reply:
Yes, so we can go with `newline=''`

Reply:
Hey guys, just came across this. I know nothing about python but it appears that the `newline` breaks everything on windows. I removed it and the execution gets through. (But should i expect an empty line every other line in the CSV?)

Reply:
Nevermind, i just saw #4.

But currently i don't have any version of python 2 on my computer, does it come with kicad itself?